code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -0,0 +1,40 @@
+/obj/item/stack/nanopaste
+	name = "nanopaste"
+	singular_name = "nanite swarm"
+	desc = "A tube of paste containing swarms of repair nanites. Very effective in repairing robotic machinery."
+	icon = 'icons/obj/nanopaste.dmi'
+	icon_state = "tube"
+	origin_tech = "materials=2;engineering=3"
+	amount = 10
+	toolspeed = 1
+
+/obj/item/stack/nanopaste/attack(mob/living/M as mob, mob/user as mob)
+	if(!istype(M) || !istype(user))
+		return 0
+	if(istype(M,/mob/living/silicon/robot))	//Repairing cyborgs
```

Comment:
I'd probably make the repair a `do_after` or something so that people aren't riding borgs into combat and smearing paste all over them constantly.

Reply:
While that does sound hilarious, you have a good point.